tox.ini

```diff
@@ -58,10 +60,9 @@ commands = python -m build
 [testenv:docs]
 description = "build docs with sphinx"
 basepython = python3.12
+dependency_groups = docs
 allowlist_externals = rm
 changedir = docs/
 # clean the build dir before rebuilding
-commands_pre =
-    pip-install-dependency-groups -f {toxinidir}/pyproject.toml docs
```

Comment:
FWIW, I left this one to keep a run with pip-install-dependency-groups. but a proper test would be better.

Reply:
I'll work up some tests, so let's not worry about it and just get to a more ideal config. (But I'll probably focus on working up `pip` support before I come back to this, since ideally the pip wrapper can just be dropped in a `2.0` release at some point.)

Reply:
Personally, I think I'd prefer a `dependency-groups` command that produced a list of deps that I could feed to `pip install` or `uv pip install` or `requirements.txt` or whatever. :)